### PR TITLE
Update ModuleManager0002Test.php

### PR DESCRIPTION
**Pull request by kshitijSharma2014**

### CODE REVIEW

**Comment by kshitijSharma2014** [comment]
@puneet0191  @javigomez  @gunjanpatel  
I have edited the code and i hope now it is following the coding standards :)

**Comment by kshitijSharma2014** [comment]
@puneet0191  just give me one more day i will remove the item exists function from my tests also.

File: tests/system/webdriver/tests/extensions/ModuleManager0002Test.php [hunk 1]
```diff
@@ -5,9 +5,14 @@
...
 /**
- * This class tests the Module Manager: Add / Edit Module Screen
- * 
+ * This class tests the  Modules: Add / Edit  Front End.
  *
+ * @package     Joomla.Tests
+ * @subpackage  Test
+ *
+ * @copyright   Copyright (coffee) 2005 - 2014 Open Source Matters, Inc.   All rights reserved.
+ * @liicense     GNU General Public License version 2 or later; see LICENSE.txt
```
Comment (gunjanpatel) [inline tests/system/webdriver/tests/extensions/ModuleManager0002Test.php]: - There is a typo in `@license`.
- `@copyright` and `@license` their comments should be align.

File: tests/system/webdriver/tests/extensions/ModuleManager0002Test.php [hunk 2]
```diff
@@ -15,16 +20,25 @@ class ModuleManager0001Test extends JoomlaWebdriverTestCase
...
 	 */
-	protected $moduleManagerPage = null; // Global configuration page
+	protected $moduleManagerPage = null;
 
+	/**
+	 * Login to back end and navigate to menu Tags.
+	 *
+	 * @since   3.0
+	 *
+	 * @return void	 *
```
Comment (gunjanpatel) [inline tests/system/webdriver/tests/extensions/ModuleManager0002Test.php]: Remove `*` from the end of the comment. It's not allowed.

File: tests/system/webdriver/tests/extensions/ModuleManager0002Test.php [hunk 3]
```diff
@@ -32,71 +46,69 @@ public function tearDown()
...
 		$this->driver->get($cfg->host . $cfg->path . $categoryManager);
 
-		$salt = rand();
-		$categoryName = 'category_ABC'.$salt;
-		$categoryName1 = 'category_ABC1_'.$salt;
-		$categoryName2 = 'category_ABC2_'.$salt;
+		$salt                      = rand();
+		$categoryName              = 'category_ABC' . $salt;
+		$categoryName1             = 'category_ABC1_' . $salt;
+		$categoryName2             = 'category_ABC2_' . $salt;
```
Comment (puneet0191) [inline tests/system/webdriver/tests/extensions/ModuleManager0002Test.php]: Why soo much space? are you using Tab?

File: tests/system/webdriver/tests/extensions/ModuleManager0002Test.php [hunk 4]
```diff
@@ -32,71 +46,69 @@ public function tearDown()
...
+
 		$this->categoryManagerPage->addCategory($categoryName);
 		$message = $this->categoryManagerPage->getAlertMessage();
 		$this->assertTrue(strpos($message, 'Category successfully saved') >= 0, 'Category save should return success');
-		
-		$parentCategory='- category_ABC'.$salt;
-		$desc="child test category";
+
+		$parentCategory = '- category_ABC' . $salt;
+		$desc           = "child test category";
```
Comment (puneet0191) [inline tests/system/webdriver/tests/extensions/ModuleManager0002Test.php]: Again!

**Comment by puneet0191** [comment]
Not merging this, because your Other PR already contains Style fixes right? for Module Manager?

**Pull request closed without merging**